charts/fluentbit/values.yaml

```diff
@@ -0,0 +1,158 @@
+namespace:
+  create : true
+  name: "calyptia-fluentbit"
+
+serviceAccount:
+  create: true
+
+image:
+  amd:
+    repository: http://709825985650.dkr.ecr.us-east-1.amazonaws.com/calyptia/calyptia-fb
```

Comment:
This is an AWS marketplace only image.
We need to consider this I think in the helm chart.

Reply:
since we changed the directory structure, i'm going to resolve this. let me know otherwise